README.md

```diff
@@ -105,6 +102,19 @@ installation is deterministic and reliable between different developer setups.
 In contrast to the `pythonpath` mode the activation of the virtual environment
 provided by [pipenv] is done automatically in the background.
 
+*NOTE:* If `pipenv` mode is configured on an OS that has a
+`/usr/bin/env` command that does not support the `-S` split string
+option, `autohooks` will automatically change to `pipenv_multiline`
+mode.
+
+### Pipenv Multi-Line Mode
+
+Identical behavior to [pipenv mode](#pipenv-mode). By using
+`pipenv__multiline` mode, the virtual environment will be activated
```

Comment:
```suggestion
`pipenv_multiline` mode, the virtual environment will be activated
```

Reply:
I think I may have done the update wrong, but this was removed as an option for the user to configure.